### PR TITLE
Feature/profile page2

### PR DESCRIPTION
**Pull request by JanPSchwarz**
for login use ceredentials:
name: feeling
password: feeling

note:
- there are many files changed due to a timeFormat change throughout the app not regarding specifically this US
- we decided to change things a little from the user story:
  -  in the profile page the user can choose all but light and dark theme
  - in the header there is a fast option button for the user to switch between the chosen theme from profile page, light and dark theme 
  - in demo-mode (not logged in) there is only dark and light 

log for main changes regarding this US:
- `profile-page.js` + `Profile` component
- backend `api-route` for user
- theme logic `POST` in `profile-page.js/Profile` and `Layout/Header`

### CODE REVIEW

File: components/Profile.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+              <StyledCircle $background={background} $text={text} />
+              <StyledInput
+                defaultChecked={currentUser.theme === name ? true : false}
+                type="radio"
+                id={name}
+                name="theme"
+                value={name}
+                $background={background}
+              />
+              <StyledLabel htmlFor={name}>{title}</StyledLabel>
```
Comment (SabinePfanner) [inline components/Profile.js:157]: I cannot find any reference input for the label?

File: pages/index.js [hunk 2]
```diff
@@ -101,6 +104,17 @@ export default function HomePage({
...
+
+  // const {
+  //   data: currentUser,
+  //   isLoading,
+  //   error,
+  //   mutate,
+  // } = useSWR(session && `/api/user/${session.user.email}`);
+
+  // if (session) getUserTheme(currentUser.theme);
+
```
Comment (SabinePfanner) [inline pages/index.js:117]: Please remove code and imports which are not used

**Review by SabinePfanner — CHANGES_REQUESTED** [review]
Great work! Just some smaller comments as I found several unused code. Additionally I would recommend to apply a consistent import logic.

**Comment by janaRicarda** [comment]
Thank your four the review! It got a little bit mixed up... i'll fix it!

File: components/Profile.js (hunk 1 above)
Comment (janaRicarda) [reply components/Profile.js:157]: the reference input is right above the label

**Review by ChristianMoschke — APPROVED** [review]
LGTM!

**Pull request merged**